packages/docs/pages/Table/TablePage.tsx

```diff
@@ -172,6 +172,50 @@ const TablePage = () => {
         {/* jsx-to-string:end */}
       </CodePreview>
 
+      <H1>Usage with TableFigure</H1>
+
+      <Text>
+        TableFigure components are used to wrap Tables and any relevant information to be grouped with them.
+        TableFigures also provide a scrollable overflow on mobile for Tables with large amounts of data. Try removing
+        the TableFigure component below in mobile view to see the differences.
```

Comment:
We don't do this on all the pages, but we should wrap mentions of components with the `<Code primary>` component:
```suggestion
        <Code primary>TableFigure</Code> components are used to wrap tables and any relevant information to be grouped with them.
        <Code primary>TableFigure</Code>s also provide a scrollable overflow on mobile for tables with large amounts of data. Try removing
        the <Code primary>TableFigure</Code> component below in mobile view to see the differences.
```

Reply:
Sure - was just trying to keep some consistency. 👍